src/css/scss/fluid-covid-map-viz.scss

```diff
@@ -518,6 +518,47 @@ button {
     padding: 0.5rem;
 }
 
+.fl-mapviz-hospital-description {
+    flex-basis: 70%;
+    border-right: 4px solid #ACACAC;
+}
+
+.fl-mapviz-hospital-description-prov {
+    @extend .fl-mapviz-hospital-prov;
+}
+
+.fl-mapviz-hospital-features {
+    flex-basis: 30%;
+}
+
+.fl-mapviz-hospital-feature {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    
+    svg {
+        flex-basis: 15%
+    }
+}
+
+.fl-mapviz-hospital-prov {
+    flex-direction: row;
+    display: flex;
+    align-items: center;
+    background: #E7E7E7;
+    height: 4rem;
+}
+
+
+.fl-mapviz-hospital-features-prov {
```

Comment:
Both `.fl-mapviz-hospital-features-prov` and `.fl-mapviz-hospital-description-prov` are extended from `.fl-mapviz-hospital-prov` without any change. Moreover, `.fl-mapviz-hospital-prov` is a css definition that is not used in the markup. Would it be easier to understand if the css for all three selectors is defined as:

```
.fl-mapviz-hospital-features-prov,
.fl-mapviz-hospital-description-prov {
    {a copy of line 545-549}
}
```